orchagent/portsorch.cpp

```diff
@@ -109,8 +110,9 @@ PortsOrch::PortsOrch(DBConnector *db, vector<string> tableNames) :
     /* Set port to hardware learn mode */
     for (i = 0; i < m_portCount; i++)
     {
-        attr.id = SAI_PORT_ATTR_FDB_LEARNING;
-        attr.value.s32 = SAI_PORT_LEARN_MODE_HW;
+        // TODO: find ??
+        // attr.id = SAI_PORT_ATTR_FDB_LEARNING;
```

Comment:
>SAI_PORT_ATTR_FDB_LEARNING [](start = 21, length = 26)

should be SAI_BRIDGE_PORT_FDB_LEARNING_MODE_HW,

but this is only available for BRIDGE PORT, meaning only valid for ports that joins dot1q bridge.

router ports are not bridge port and do not need such option.